### ISSUE TITLE
Incorrect colors (too little contrast) on html messages in message pane

### ISSUE THREAD

**Issue by lstandish** [thunderbird-conversations/thunderbird-conversations#2071]
Thunderbird version: 115.3.1
Thunderbird Conversation version: 4.1.5
Operating System: Linux Debian 11

What did you do?
Selected an html message, such as a newsletter.

What did you expect to happen?
Message should be displayed in the message pane with the same  background and text colors as before installing Thunderbird Conversation.

What actually happened?
After installing Thunderbird Conversations, html messages from many sources (such as substack.com) display in the message pane with a very dark background color and dark grey or grey text color, resulting in too little contrast for comfortable viewing:

![Screenshot_2024-01-07_23-09-15](https://github.com/thunderbird-conversations/thunderbird-conversations/assets/5531047/023713f3-decd-4520-8801-612a6cb00c62)

Thanks so much for this great extension!

**Comment by Standard8** [comment]
This sounds like you have some custom colour settings or something that's changing the background of the email. Conversations doesn't directly change the colours of emails, and I've not seen this before, so I would at least need to know if there have been any settings changed and maybe an example email.

**Comment by lstandish** [comment]
Hello and thank you for your reply.  I think you are right that something has changed the "background color" of my html mail from white (the default, I believe) to a very dark color.  The strange thing is that this is only seen when I have the Conversations extension enabled.  When I disable the extension, the background color returns to normal.  This of course is why I assumed it must be caused by the Conversations extension.

I have not changed any colors that I can remember (I instead have installed themes), but today I went to Settings/Composition to check on that.  Under HTML Style, I have checked "Use reader's default colors."  I also clicked the "Restore Defaults" button for good measure.  I also closed and restarted Thunderbird.

As far as themes go, I had installed Dark theme (by Mozilla, version 1.3), and it is the theme I would prefer to use with Conversations, but for test purposes, I disabled it and just have the default "System theme" (the one in use when no other theme is enabled).  Even with just the default theme active, I get this background color display problem for html mails whenever Conversations is enabled (and only when it is enabled).  

It is an html email display problem only.  Html messages composed in Thunderbird do not contain any incorrect background styling.

I think this must have something to do with my having previously enabled dark theme.  I think somehow the dark theme background persists (but only when Conversations is enabled) even though dark theme is no longer enabled.

**Comment by lstandish** [comment]
Hi, and thanks for the response.  I replied to your comment yesterday, but somehow it was not properly posted.

I don't have any custom color settings - at least, none that I can access.  I checked for that (Edit/Settings/Composition/HTML Style). "Use reader's default colors" is checked (I also tried clicking "Restore Defaults.")  I also tried restarting Thunderbird after doing this.

I use "dark" theme.  This dark background color (for html mail in the preview panel) makes me suspect it has to do with this dark theme.  However, the problem is NOT seen when dark theme is enabled, as long as Conversations extension is disabled.  However, if Conversations is enabled, it is seen even when I revert to default ("system") theme, and disappears when I remove the Conversations extension.

I have observed something that might help.  The dark background color appears to be the same color that dark theme applies for the text-mode (not html) preview of collapsed threads.  I am attaching a screenshot to illustrate that.
![Screenshot_2024-01-21_10-39-13](https://github.com/thunderbird-conversations/thunderbird-conversations/assets/5531047/c4f3a2ce-28bc-40bc-b15f-d3ea135be914)
